gnucash/report/standard-reports/transaction.scm

```diff
@@ -907,8 +907,7 @@ Credit Card, and Income accounts."))))))
                               (begin
                                 (if column-amount
                                     (set! merging-subtotal
-                                          (merge-fn merging-subtotal column-amount
-                                                    GNC-DENOM-AUTO GNC-RND-ROUND)))
+                                          (merge-fn merging-subtotal column-amount)))
```

Comment:
This is not an "old comment removed" and I'm still not sure why you chose to change the rounding strategy. So your commit message is a bit lacking here/your commit mixes two separate topics.

Reply:
Hmm because I discovered gnc-numeric-add-fixed and thought it was rather underused :) ... and if I strictly use additions/subtractions then the rounding strategy will always be irrelevant? IIRC the rounding errors tend to happen when calculating taxes which involve multiplication. I haven't looked deeply into the rounding options but my guess is they're all harmless when adding numbers?

Reply:
>  I haven't looked deeply into the rounding options but my guess is they're all harmless when adding numbers?

Nope. GncNumeric is a rational number. As long as the denominators are the same then there's rounding only on very large numbers, but if the denominators differ and don't have smallish common factors then rounding can become significant.

Reply:
We're adding the Debit/Credit subtotal numbers here. So best to leave it as GNC-DENOM-AUTO GNC-RND-ROUND?